js/controller.js

```diff
@@ -269,7 +269,10 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         element = elementVideo;
```

Comment:
I've already talked to Angie about it. I don't recall what the final outcome was, but there is already a ticket for why it's not instantiated.  So this checkin should just be concerned with what to do if we get bad data and being able to fail gracefully.